stories/flow-core/f-pictogram.stories.js

```diff
@@ -157,6 +164,46 @@ export const Variant = {
 	}
 };
 
+export const Category = {
+	render: args =>
+		html`<f-div gap="medium" padding="x-large" direction="row" align="middle-center">
```

Comment:
Can you convert this file to TS, we have a lot of stories in JS but as we move let's convert it all slowly